### ISSUE TITLE
Links within help text - background

### ISSUE THREAD

**Issue by Jay-Neighbours-I72** [truetandem/e-QIP-prototype#1169]
- [x] Add a white background to links within help text to pass WCAG AA

<img width="897" alt="screen shot 2017-05-08 at 3 08 50 pm" src="https://cloud.githubusercontent.com/assets/19935974/25820746/5f2a0a72-3400-11e7-8d27-96a8fb9d0b99.png">
![screen shot 2017-05-08 at 3 08 59 pm](https://cloud.githubusercontent.com/assets/19935974/25820750/607510d4-3400-11e7-9502-efa8f2e2b55a.png)

**Comment by bmallred** [comment]
## Help message
![message-anchor](https://cloud.githubusercontent.com/assets/697855/26014838/58ca3baa-372c-11e7-9ce6-170c91d1a331.png)

## Error message
![message-anchor2](https://cloud.githubusercontent.com/assets/697855/26014861/77707dda-372c-11e7-94f9-41565a697b8d.png)

cc: @Jay-Neighbours-I72 @lauraponce

**Comment by lauraponce** [comment]
Is it possible to have the white background only extend to where there is text to avoid this excess white area? @bmallred 

![screen shot 2017-05-16 at 11 59 06 am](https://cloud.githubusercontent.com/assets/21222704/26115800/15e62136-3a2f-11e7-826f-944b608c6873.png)

**Comment by bmallred** [comment]
@lauraponce this actually has changed in PR #1236. Should be up there in the next hour! I'll move it back in to review when that is up there.

**Comment by billy22g** [comment]
What's the desired state for this @bmallred? I'm still seeing:
<img width="671" alt="screen shot 2017-05-30 at 7 40 14 am" src="https://cloud.githubusercontent.com/assets/5091167/26586061/40e15bbc-450b-11e7-9ff5-f0485633e253.png">

**Comment by Jay-Neighbours-I72** [comment]
@billy22g Yeah this is the desired state (round 2)

First idea was the white background, it was driving me crazy. Demanding way more attention than necessary and competing with the help text itself.

So instead we went with a darker link color that will still pass WGAC AA on the light blue bg

**Comment by billy22g** [comment]
Cool, LGTM. Let's just try to document that in the future if the desired state changes from the initial issue. Thanks y'all!
